wifiphisher/common/interfaces.py

```diff
@@ -492,10 +493,7 @@ def set_interface_mac(self, interface_name, mac_address):
             pyw.macset(card, mac_address)
         # make sure to catch an invalid mac address
         except pyric.error as error:
-            if error[0] == 22:
-                raise InvalidMacAddressError(mac_address)
-            else:
-                raise
+            raise InvalidMacAddressError(mac_address)
 
```

Comment:
In my system if I set the mac address to `11:11:11:11:11:11`, pyric will raise error code 99.